notes/returned-mail-received.vbs

```diff
@@ -307,65 +333,73 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
         		cancel_without_confirmation
         		If isdate(date_requested) = FALSE THEN  err_msg = err_msg & vbnewline & "Please enter the date verifications were requested."
 				IF Cdate(date_requested) > cdate(date) = TRUE THEN  err_msg = err_msg & vbnewline & "You must enter an actual date that is not in the future."
-        		IF ECF_reveiwed = "Select One:" THEN  err_msg = err_msg & vbnewline & "Please review ECF to ensure the requested verifications are not on file."
-        		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
-        	LOOP UNTIL err_msg = ""
+        		IF ECF_reveiwed = "Select:" THEN  err_msg = err_msg & vbnewline & "Please review ECF to ensure the requested verifications are not on file."
+	    	LOOP UNTIL err_msg = ""
         	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to      password back into MAXIS
         LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 
         'per POLI/TEMP this only pertains to active cash and snap '
-        IF case_active = TRUE THEN
+        IF snap_or_cash_case = TRUE THEN
         	CALL MAXIS_background_check
-        	CALL navigate_to_MAXIS_screen("STAT", "PACT")
-        	'Checking to see if the PACT panel is empty, if not it create a new panel'
+			Dialog1 = "" 'Running the initial dialog
+			BeginDialog Dialog1, 0, 0, 351, 65, "Returned Mail Processing PACT panel entry"
+			  ButtonGroup ButtonPressed
+			    OkButton 240, 45, 50, 15
+			    CancelButton 295, 45, 50, 15
+			    PushButton 5, 45, 65, 15,  "PACT TE02.13.10", POLI_TEMP_PACT_button
+			  Text 5, 5, 340, 20, "When mail to the client has been returned to the agency and the agency has sent a Request for Contact to the client with no response, for SNAP programs used code 4 will be used and code 3 for CASH."
+			  Text 5, 30, 305, 15, " Allow 10 days for the client to respond to the Verification Request before terminating benefits. "
+			EndDialog
+
+			DO
+        		err_msg = ""
+        		DIALOG Dialog1
+        		cancel_without_confirmation
```

Comment:
This should have a confirmation - we are pretty far in the script to lose everything.